<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1643</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skribisto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons/skr_icons.qrc">
    <normaloff>:/icons/skribisto/skribisto.svg</normaloff>:/icons/skribisto/skribisto.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1643</width>
     <height>37</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionBack_up"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose_project"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAdd_Window"/>
    <addaction name="actionShow_Project_Dock"/>
    <addaction name="actionShow_View_Dock"/>
    <addaction name="separator"/>
    <addaction name="actionClose_all_views"/>
    <addaction name="separator"/>
    <addaction name="actionSwitch_theme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Dock" name="projectDock">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>524287</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="Dock" name="viewDock">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>524287</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="bottomDock">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_project">
   <property name="icon">
    <iconset theme="applications-office">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAdd_Window">
   <property name="text">
    <string>Add Window</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDisctraction_Free">
   <property name="text">
    <string>Disctraction Free</string>
   </property>
   <property name="toolTip">
    <string>Switch to disctraction free mode</string>
   </property>
  </action>
  <action name="actionShow_View_Dock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/icons/skr_icons.qrc">
     <normaloff>:/icons/skribisto/dock-right-close.svg</normaloff>:/icons/skribisto/dock-right-close.svg</iconset>
   </property>
   <property name="text">
    <string>Show View Dock</string>
   </property>
  </action>
  <action name="actionShow_Project_Dock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/icons/skr_icons.qrc">
     <normaloff>:/icons/skribisto/dock-left-close.svg</normaloff>:/icons/skribisto/dock-left-close.svg</iconset>
   </property>
   <property name="text">
    <string>Show Project Dock</string>
   </property>
  </action>
  <action name="actionClose_all_views">
   <property name="text">
    <string>Remove All Splits</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionBack_up">
   <property name="text">
    <string>Back up</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSwitch_theme">
   <property name="icon">
    <iconset resource="../../3rdparty/icons.qrc">
     <normaloff>:/icons/backup/whitebalance.svg</normaloff>:/icons/backup/whitebalance.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">Switch to light theme</string>
   </property>
   <property name="iconText">
    <string notr="true">Switch to light theme</string>
   </property>
   <property name="toolTip">
    <string notr="true">Switch to light theme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dock</class>
   <extends>QDockWidget</extends>
   <header>dock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../3rdparty/icons.qrc"/>
  <include location="../../resources/icons/skr_icons.qrc"/>
  <include location="../../resources/icons/skr_icons.qrc"/>
 </resources>
 <connections/>
</ui>
